src/requirements/Uniswap/UniswapForm.tsx

```diff
@@ -50,21 +54,26 @@ const UniswapForm = ({
 
   const [shouldShowUrlInput, setShouldShowUrlInput] = useState(false)
 
-  const chain: (typeof consts.UniswapV3PositionsChains)[number] = useWatch({
+  const chain: UniswapChains = useWatch({
```

Comment:
Wrote a comment about a better solution for these types somewhere, the same applies here ;)